### PR TITLE
Add docker builds and fix release github action

### PR DESCRIPTION
**Pull request by MSevey**
# PULL REQUEST

## Overview
Adds docker build GitHub action and Fix release action.

## Issues Closed

Closes SKY-470

### CODE REVIEW

File: .github/workflows/ci_release.yml [hunk 1]
```diff
@@ -62,10 +62,28 @@ jobs:
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+
+      - uses: SkynetLabs/.github/.github/actions/docker-publish@master
+        with:
+          docker_username: ${{ secrets.DOCKER_USERNAME}}
+          docker_password: ${{ secrets.DOCKER_PASSWORD}}
+          docker_repository: skynetlabs/skynet-accounts
```
Comment (ro-tex) [inline .github/workflows/ci_release.yml:88]:
```suggestion
          docker_repository: skynetlabs/pinner
```

**Review by ro-tex — DISMISSED** [review]
LGTM.

File: .github/workflows/ci_release.yml [hunk 2]
```diff
@@ -62,10 +62,28 @@ jobs:
     needs: changes
     runs-on: ubuntu-latest
     if: ${{ (needs.changes.outputs.updates == 'true' && github.event_name == 'schedule') || github.event_name == 'workflow_dispatch' }}
+    outputs:
+      new_version: ${{ steps.version.outputs.new-version }}
```
Comment (kwypchlo) [inline .github/workflows/ci_release.yml:66]: have you checked whether dashed property works with dot accessor ? in javascript that would not work and you would need to do sth like `steps.version.outputs["new-version"]` so I'm just concerned and you're also assigning it to a variable with a underscore which also raises my concern :)

**Review by kwypchlo — CHANGES_REQUESTED** [review]

File: .github/workflows/ci_release.yml (hunk 2 above)
Comment (MSevey) [reply .github/workflows/ci_release.yml:66]: yup, this was all tested on the docker test repo. All these PRs are not just copying the same code to all the repos. 

the `-` vs `_` doesn't matter. That is just mis matched conversations between actions. 

I tried to stay consistent in the actions themselves with how the external actions are set up. But then that leads to the inevitable mismatch for things like this. Ultimately there is no way around it, there will be mismatches somewhere. 

yml doesn't care if the variable has a `-` or a `_`

File: .github/workflows/ci_release.yml [hunk 3]
```diff
@@ -62,10 +62,28 @@ jobs:
...
+          version-bump: ${{github.event.inputs.version}}
+
+  docker:
+    needs: release
+    name: Build and Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+
```
[the remark this answers is not in the document]
Comment (MSevey) [reply .github/workflows/ci_release.yml:83]: you are right, we can remove this duplicate.

**Review by ro-tex — APPROVED** [review]

**Pull request merged**